index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width= , initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css" integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="./style/main.css">
+    <title>KMarketer</title>
+</head>
+<body>
+    <div class="content">
+        <div class="header header--bg-gray">
+            <div class="header__item ">
+                <div class="logo">
+                    <h1 class="">KMarketer</h1>
+                </div>
+                <ul class="navigation">
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Home</a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Product</a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Pricing</a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">Contact</a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">
+                            <img src="./assets/svgs/search.svg" alt="">
+                        </a>
+                    </li>
+                    <li class="navigation__item">
+                        <a class="link link--disabled" href="#">
+                            <img src="./assets/svgs/cat.svg" alt="">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <div class="header__content">
+                <h1 class="site-header site-header--large " >
```

Comment:
`site-header site-header--large` those class are global I guess, isn't there a way to avoid them. cause coming to meaning, this does not means something in the below code.